logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/client/NeedsToRegisterClientUseCase.kt

```diff
@@ -17,7 +17,7 @@ class NeedsToRegisterClientUseCaseImpl(
 ) : NeedsToRegisterClientUseCase {
     override suspend fun invoke(): Boolean =
         sessionRepository.userAccountInfo(selfUserId).fold(
-            { true },
+            { false },
```

Comment:
This will break thee app
if you log in and does not finish registering a client, then the app will not navigate to the add new device screen

Reply:
nvm i was wrong